packages/core/contracts/financial-templates/common/FundingRateApplier.sol

```diff
@@ -228,18 +228,18 @@ abstract contract FundingRateApplier is FeePayer {
                     fundingRate.updateTime = proposalTime;
 
                     // If there was no dispute, send a reward.
+                    FixedPoint.Unsigned memory reward = FixedPoint.fromUnscaledUint(0);
                     OptimisticOracleInterface.Request memory request =
                         optimisticOracle.getRequest(address(this), identifier, proposalTime, ancillaryData);
                     if (request.disputer == address(0)) {
-                        FixedPoint.Unsigned memory reward =
-                            _pfc().mul(_getConfig().rewardRatePerSecond).mul(proposalTime.sub(lastUpdateTime));
+                        reward = _pfc().mul(_getConfig().rewardRatePerSecond).mul(proposalTime.sub(lastUpdateTime));
                         if (reward.isGreaterThan(0)) {
                             _adjustCumulativeFeeMultiplier(reward, _pfc());
                             collateralCurrency.safeTransfer(request.proposer, reward.rawValue);
                         }
                     }
 
-                    emit FundingRateUpdated(fundingRate.rate.rawValue, fundingRate.updateTime);
+                    emit FundingRateUpdated(fundingRate.rate.rawValue, fundingRate.updateTime, reward.rawValue);
```

Comment:
As highlighted in the PR description, do we think there's any reason to also include the `currentTime` in this event? i.e. the timestamp at which the proposal was published?

Reply:
Emitted logs _do_ include timestamps but we've historically included time for sensitive things in other places (such as the DVM) so I don't really see any downside in including it. However, I think the issue from OZ also wants the `proposalTime` within the event, which is currently not there.

Reply:
Agreed. Proposal time might be useful to identify the request.

Reply:
The thing is, this event only fires once a new proposal is published. So the `updateTime` is always set to the `proposalTime`. So the `proposalTime` is always emitted, but its just equivalent to the `updateTime` now.

Reply:
Ahhh good point.

Reply:
in that case, then it's good then. perhaps add a comment to explain that?

Reply:
Done ptal